picard/ui/options/metadata.py

```diff
@@ -97,6 +103,16 @@ def load(self):
             if locale == current_locale:
                 combo_box.setCurrentIndex(i)
 
+        self.ui.translate_artist_names_exception.setChecked(config.setting["translate_artist_names_exception"])
+
+        combo_box = self.ui.artist_locale_exception
+        current_locale = config.setting["artist_locale_exception"]
+        for i, (locale, name, level) in enumerate(iter_sorted_locales(CHAR_SETS)):
+            label = name
+            combo_box.addItem(label, locale)
```

Comment:
Could this be shortened to:

```
combo_box.addItem(name, locale)
```

Unless I'm missing something, `label` isn't used anywhere else.